src/Api/ApigeeX/Controller/ListingHelperTrait.php

```diff
@@ -54,6 +55,37 @@ protected function getRawList(UriInterface $uri): array
         return reset($responseArray);
     }
 
+    /**
+     * Returns a raw API response as an array when there is single value in api response.
+     *
+     * @param \Psr\Http\Message\UriInterface $uri
+     *   URI of the endpoint where the request should be sent.
+     *
+     * @return array
+     *   API response as an array.
+     */
+    protected function getRawSingleList(UriInterface $uri): array
+    {
+        $response = $this->getClient()->get($uri);
+        $responseArray = $this->responseToArray($response);
+
+        return $responseArray;
+    }
+
+    /**
+     * Returns an array of supported currency.
+     *
+     * @param array $currencyArray
+     *   Supported currency array
+     *
+     * @return array
+     *   Values as an array.
+     */
+    protected function getdefaultCurrency($currencyArray): array
```

Comment:
Nitpick: `getDefaultCurrency`, but this interface is used by other entity classes so this method which is only used by SuppertedCurrency controller must not be here.